app/src/main/java/com/github/gotify/Utils.java

```diff
@@ -77,6 +78,20 @@ public void onPrepareLoad(Drawable placeHolderDrawable) {}
         };
     }
 
+    public static String binaryFileToBase64(@NonNull InputStream inputStream) {
```

Comment:
This method needs a `@RequiresApi` annotation, see build error.

The usage of this method then must be wrapped into an if inside `com.github.gotify.login.LoginActivity#onActivityResult`.